packages/bootstrap/README.md

```diff
@@ -1,50 +1,136 @@
+$grid-border-width: 2px;
+$grid-border: #000000;
+
+// When you are finished with your customizatoni, import the theme
+@import "@progress/kendo-theme-bootstrap/dist/all.scss";
 
-The handling of all variables happens in the [`_bootstrap-map.scss`](src/_bootstrap-map.scss) file. It loads the Bootstrap variables from the default location. For example, `node_modules/bootstrap/scss/_variables.scss`.
+// Note: the variables bellow this point will not take effect,
+// because the theme is already imported
 
-If you have a customized version of Bootstrap:
+$grid-border: red;
 
-1. Specify the location in `_bootstrap-map.scss`.
-1. Build as usual.
+```
 
-## Applying Mapped Variables
+## Integrating with third party frameworks
 
-To ensure a certain level of compatibility, the Bootstrap variables are locally mapped. This also enables Kendo UI to use an ad-hoc micro framework based on primitive components. For example, [button](src/mixins/appearance/_button.scss), [input](src/mixins/appearance/_input.scss), [node](src/mixins/appearance/_node.scss), and so on.
+CSS can be complicated due to cascades (that's what C stands for). If there are multiple frameworks loaded on a single page, it is quite possible that their styles will interfere with one another.
 
-However, some variables, such as `$tooltip-arrow-width` which is used in the tooltip, are not mapped. They are referenced explicitly because they do not fit in the previously mentioned framework.
+To avoid that, we've prefixed all our class names with `k-` e.g. `k-grid`, `k-hidden` etc., with the exception of handling for `hidden` attribute, which will hide the element that has it.
+
+Some frameworks, like Bootstrap and Tailwind, touch global styles to establish a sort of baseline -- we call that normalizing. One such styles is the following bit, which makes dimenssions (width and height) behave in a specific way:
+
+```css
+*, ::after, ::before {
+    box-sizing: border-box;
+}
+```
+
+We've ensured that our themes work correctly with both those styles being present and absent.
+
+However, even when being cautious, neglecting the scope and cascade of CSS, that may result in unexpected side effects. Oddly enough, most of the cases when that happens, everything is expected, at least from CSS point of view, but it can be still frustrating.
+
+For instance, it cannot be expected that adding `form-control` from Bootstrap to a Kendo UI Combobox will make the component behave and look like a Bootstrap form component. The scope and cascade are different -- `form-control` is intended to be added to pure `input` elements, where as a Kendo UI Combobox has a different structure and styles will not be applied correctly.
+
+Another example is adding `bg-red-400` from Tailwind to a Kendo UI Button, depending on the exact stylesheet order that class may or may not be applied.
+
+As a general rule, it should not be expected that mixing at matching classes from different frameworks will yield the desired results.
+
```

Comment:
Can we add something about our swatches for the Bootstrap theme?

Reply:
This is the readme. the one that's going to be shown in npm. it's more of an overview than a hyper detailed guide